src/inspect_ai/_util/file.py

```diff
@@ -77,6 +79,23 @@ def file(
             f.close()
 
 
+def basename(file: str) -> str:
```

Comment:
It would be nice to have unit tests of this function with s3, local Windows, etc. filenames

Reply:
Just added these.